gru_language_model.py

```diff
@@ -0,0 +1,223 @@
+import os
```

Comment:
This file is just a sample code we should use for copying and pasting in repo development. I would put it in a separate folder so that future users wouldn't get confused.

Reply:
This hasn't been removed, as far as I can tell

Reply:
Apologies, must have missed this comment